.gitlab/build_quartz.yml

```diff
@@ -70,11 +70,11 @@ quartz-gcc_8_3_1-src:
 quartz-clang_10_0_0-full:
   variables:
     COMPILER: "clang@10.0.0"
-    SPEC: "%${COMPILER}"
+    SPEC: "--spec=%${COMPILER}"
```

Comment:
Pulled the commandline argument into this variable so it could be empty in lassen.